scripts/serve.js

```diff
@@ -15,6 +15,23 @@ const devMiddleware = webpackDevMiddleware(compiler, {
 app.use(webpackHotMiddleware(compiler))
 app.use(devMiddleware)
 
+/* Hardcoded mock data for deployments */
+app.get('/api/deployments', function(req, res) {
```

Comment:
So the global tracker service already has build data available for you to use (deployments), as I explained in the Skype chat it might be easier to just have the the endpoint return mocked service data so we're using the same data structure across our entire application. Just update the tracker service's existing `update` method to fetch from your given endpoint.

There's also a method in there for searching for service builds. You can toss more services in there by name by just adding onto the provided array, and it should spit up all the data you need.

Reply:
I'm sorry if I made the data structure a bit ambigous, each service object has an array of "builds" that correlate to the deployments.

Reply:
Do you mind adding a mock service object in there? I didn't immediately understand that everything was included, I'll definitely change the mock API to return that structure then and rewrite my parts to use tracker

Reply:
You should be able to import the generator method in src/util/mock which returns a randomized service object.

Reply:
There's also an example in there of the data structure.